supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -661,7 +661,8 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
         case _: RunStatus.Success if returnCode.isFailure =>
           FailedNonRetryableExecutionHandle(new Throwable(s"execution failed: could not parse return code as integer: " + returnCodeContents.get)).future
         case _: RunStatus.Success if !continueOnReturnCode.continueFor(returnCode.get) =>
-          FailedNonRetryableExecutionHandle(new Throwable(s"execution failed: disallowed command return code: " + returnCode.get), returnCode.toOption).future
+          val badReturnCodeMessage = s"Call ${call.fullyQualifiedName}: " + s"return code was ${returnCode.getOrElse("(none)")}"
+          FailedNonRetryableExecutionHandle(new RuntimeException(badReturnCodeMessage), returnCode.toOption).future
```

Comment:
oops, I didn't edit that correctly.

2016-08-24 14:38:11,256 cromwell-system-akka.dispatchers.engine-dispatcher-89 ERROR - WorkflowManagerActor Workflow 75867985-a777-486e-ab59-882e4841a47d failed (during ExecutingWorkflowState): java.lang.Exception: Call invalid_return_code_wf.invalid_return_code: return code was 0